src/components/profile/ProjectItem.tsx

```
@@ -42,85 +47,96 @@ export default function ProjectItem({
+            {/* 수정 및 삭제 버튼 - 아래로 이동 */}
+            {isMy && (
+              <div className="flex gap-4 mt-4 justify-end">
+                <button
+                  className="flex items-center gap-2 px-4 py-2 bg-main-color text-white rounded-lg hover:bg-secondary-color-dark transition-colors w-full justify-center"
+                  onClick={handleEditClick}
+                >
+                  <Pencil className="h-5 w-5" />
+                  <span>수정</span>
+                </button>
+                <button
+                  className="flex items-center gap-2 px-4 py-2 border border-gray-300 text-sub-gray rounded-lg hover:bg-gray-100 transition-colors w-full justify-center"
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)